tests/vm_util_test.py

```diff
@@ -17,10 +17,56 @@
 import unittest
 
 import mock
+import os
 
 from perfkitbenchmarker import vm_util
 
 
+if os.name is 'posix':
```

Comment:
Sounds good.

On Tue, May 5, 2015 at 8:48 AM, Carlos Torres <notifications@github.com>
wrote:

> In tests/vm_util_test.py
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/231#discussion_r29682144>
> :
>
> >
> >  from perfkitbenchmarker import vm_util
> >
> >
> > +if os.name is 'posix':
>
> I think I'm just going to drop the tests, even if I test for making sure
> subprocess.Popen gets the right argument, I wouldn't be testing any new
> logic, it will just be a "variable assignment".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/231/files#r29682144>
> .
>



-- 
Connor McCoy | Software Engineer | connormccoy@google.com | 206-876-1657